client/src/components/CardInfo/CardInfo.jsx

```diff
@@ -0,0 +1,29 @@
+import "../CardInfo/CardInfo.css";
+
+const CardInfo = () => {
+  return (
+    <div>
+      <h3>Team members</h3>
+      <aside>
+        <table>
+          <tr>
```

Comment:
I wonder, should we <thead> to show where the form headers are? Like below...?   Just semantics

<thead>
            <tr>
              <th>Members</th>
              <th>Percentage</th>
              <th>PRs</th>
              <th>Difference</th>
            </tr>
          </thead>`